lexicons/app/bsky/unspecced/searchActorsSkeleton.json

```diff
@@ -0,0 +1,56 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.unspecced.searchActorsSkeleton",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Backend Actors (profile) search, returning only skeleton",
+      "parameters": {
+        "type": "params",
+        "required": ["q"],
+        "properties": {
+          "q": {
+            "type": "string",
+            "description": "search query string; syntax, phrase, boolean, and faceting is unspecified, but Lucene query syntax is recommended. For typeahead search, only simple term match is supported, not full syntax"
+          },
+          "typeahead": {
+            "type": "boolean",
+            "description": "if true, acts as fast/simple 'typeahead' query"
+          },
+          "limit": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 100,
+            "default": 25
+          },
+          "cursor": {
+            "type": "string",
+            "description": "optional pagination mechanism; may not necessarily allow scrolling through entire result set"
+          }
```

Comment:
In the past it's proven useful to not support a cursor for typeahead search.  Just wanted to note that in case it's relevant here, since both `typeahead` and `cursor` parameters can appear together.

Reply:
Would the alternative be a second endpoint just for typeahead? That feels expansive to me. I think in typeahead mode we can safely ignore the `cursor` param and never populate that field in the output.